.github/CODEOWNERS

```diff
@@ -69,6 +69,24 @@
 #CC# /x-pack/plugins/drilldowns/ @elastic/kibana-app-services
 #CC# /packages/kbn-interpreter/ @elastic/kibana-app-services
 
+### Observability Plugins
+
+# Observability Shared
+/x-pack/plugins/observability/ @elastic/observability-ui
+
+# Logs and Metrics (Infra)
+/x-pack/plugins/infra/ @elastic/logs-metrics-ui
+/x-pack/plugins/infra/server/routes/log* @elastic/logs-metrics-ui @elastic/logs-ui
+/x-pack/plugins/infra/public/pages/logs/ @elastic/logs-metrics-ui @elastic/logs-ui
+/x-pack/plugins/infra/server/routes/metrics* @elastic/logs-metrics-ui @elastic/metrics-ui
+/x-pack/plugins/infra/public/pages/metrics/ @elastic/logs-metrics-ui @elastic/metrics-ui
+
+# Stack Monitoring
+/x-pack/plugins/monitoring/ @elastic/stack-monitoring-ui @elastic/logs-metrics-ui @elastic/metrics-ui
```

Comment:
This is the main change: adding @elastic/logs-metrics-ui and @elastic/metrics-ui as CODEOWNERS to the monitoring plugin (but I left @elastic/stack-monitoring-ui there as well, for clarity) One review from any metrics-ui member will satisfy all those criteria.